python/core/qgsexpression.sip

```diff
@@ -194,9 +194,9 @@ class QgsExpression
     class Function
     {
       public:
-        Function( QString fnname, int params, QString group, QString helpText = QString(), bool usesGeometry = false, QStringList referencedColumns = QStringList() );
+        Function( const QString& fnname, int params, const QString& group, const QString& helpText = QString(), bool usesGeometry = false, QStringList referencedColumns = QStringList(), bool lazyEval = false );
         /** The name of the function. */
-        QString name();
+        const QString& name();
```

Comment:
Note: returning implicitly shared classes (like QString, QVariant, QList etc) by value is IMHO better. Qt API does that all the time, and it leaves you some extra flexibility: one can return a temporary instance.